js/tests/unit/dom/manipulator.spec.js

```diff
@@ -73,39 +75,40 @@ describe('Manipulator', () => {
   })
 
   describe('getDataAttribute', () => {
```

Comment:
can you add a test here with a non bs attribute please ? To check it doesn't get non bs attribute

Reply:
Added @Johann-S 👍